src/Multilingual-Languages/LanguageEnvironment.class.st

```diff
@@ -85,12 +88,13 @@ LanguageEnvironment class >> fileNameConverterClass [
 LanguageEnvironment class >> initKnownEnvironments [
 	"LanguageEnvironment initKnownEnvironments"
 
-	| known |
+	| known localeIdClass |
+	localeIdClass := self localeIdClass.
```

Comment:
Now there is `localeIdClass` method, storing in a variable is a bit redundant. 
Really a minor thing and I approve this PR with or without changing that.

Reply:
I don't mind changing it but since it is used inside a loop it seemed appropriate to cache it.

Reply:
That is a fair policy. Leave it as is.